client/oci_env/utils.py

```diff
@@ -351,7 +351,8 @@ def _service_containers():
             )
 
         # List all containers that match the PROJECT_NAME pattern. e.g: oci_env
-        cmd = [binary, "ps", "-q", "--filter", f"name={project_name}", "--format", "{{.Names}}"]
+        #   WARNING: Ignoring custom format, because both --format and --quiet are set.
+        cmd = [binary, "ps", "--filter", f"name={project_name}", "--format", "{{.Names}}"]
```

Comment:
curious... you say "--quiet" is set, but `-q` was removed from the command, is that intentional?

Reply:
Whoever wrote this command didn't realize that -q causes the command to output just the container IDs, wheres the --format is telling it to print out the names. On docker, you end up with just the IDs because the format is ignored and then the filtering doesn't work and your stack doesn't come up (at least in newer docker sometimes). Docker also tells you that the arguments are mutually exclusive, whereas podman just pretends everything is fine. The comment I added is the error from docker so that people wouldn't tell me i'm crazy.